### PR TITLE
Remove paskino channel requirement

### PR DESCRIPTION
**Pull request by lauramurgatroyd**
Now that eqt is on conda-forge we do not need the paskino channel.

### CODE REVIEW

File: recipe/dev_environment.yml [hunk 1]
```diff
@@ -1,7 +1,6 @@
-name: idvc_dev
+name: idvc_dev2
```
Comment (lauramurgatroyd) [inline recipe/dev_environment.yml:1]:
```suggestion
name: idvc_dev
```

File: src/idvc/idvc.py [hunk 2]
```diff
@@ -3,6 +3,7 @@
 import os, sys
 
 def main():
+    print("Running main")
```
Comment (lauramurgatroyd) [inline src/idvc/idvc.py:6]:
```suggestion
```

**Comment by casperdcl** [comment]
ok the `mambabuild` output is slightly less verbose than `conda build`. Running the latter locally I get:

```
The following packages are incompatible
├─ ccpi-dvc >=22.0.0  is not installable because it requires
│  └─ _openmp_mutex >=5.1 , which conflicts with any installable versions previously reported;
├─ ccpi-viewer >=22.4.0  is installable with the potential options
│  ├─ ccpi-viewer 22.4.0 would require
│  │  ├─ eqt, which requires
│  │  │  └─ python >=3.7  with the potential options
│  │  │     └─ python [3.12|...|3.7], which can be installed;
│  │  ├─ python 3.6.* , which conflicts with any installable versions previously reported;
│  │  └─ python_abi 3.6.* *_cp36m, which conflicts with any installable versions previously reported;
│  └─ ccpi-viewer [22.4.0|23.0.0|23.1.0] would require
│     └─ python 3.10.* , which can be installed;
└─ vtk 8.1.2.*  is not installable because there are no viable options
   ├─ vtk 8.1.2 would require
   │  └─ python >=2.7,<2.8.0a0 , which conflicts with any installable versions previously reported;
   ├─ vtk 8.1.2 would require
   │  └─ python >=3.6,<3.7.0a0 , which conflicts with any installable versions previously reported;
   └─ vtk 8.1.2 would require
      └─ python >=3.7,<3.8.0a0  but there are no viable options
         ├─ python [3.7.0|3.7.1|3.7.2|3.7.3|3.7.6], which cannot be installed (as previously explained);
         ├─ python [3.7.0|3.7.1|...|3.7.9] conflicts with any installable versions previously reported;
         └─ python [3.7.10|3.7.12|...|3.7.9], which cannot be installed (as previously explained).
```

**Comment by lauramurgatroyd** [comment]
We agreed to close this for now until we are able to support VTK >8.1.2

**Pull request closed without merging**